cs251tk/toolkit/args.py

```diff
@@ -28,13 +28,15 @@ def build_argparser():
     parser.add_argument('--skip-update-check', action='store_true',
                         default=getenv('CS251TK_SKIP_UPDATE_CHECK', False) is not False,
                         help='skips the pypi update check')
+    parser.add_argument('--ci', action='store_true',
+                        help='Configure for gitlab-ci usage')
 
     specs = parser.add_argument_group('control the homework specs')
-    specs.add_argument('--course', default='sd', choices=['sd', 'hd', 'ads'],
+    specs.add_argument('--course', default='sd',
```

Comment:
I think it's still be nice to list the allowed courses? The choices list is printed when you give --help.

Reply:
The problem is that it throws an exception if I try to use `--course sd-s19`, for example. If we can somehow allow those too, then that would work.

Reply:
Ah. How about we put the three into the help string, then, along with an example of sd-s19?